app/change_sets/hyrax/work_change_set.rb

```diff
@@ -123,8 +129,27 @@ def work_members_json
       end.to_json
     end
 
+    def permissions_changed?
+      changed.include?('edit_users') ||
```

Comment:
I don't feel strongly about this, but what would you feel about using `#any?` here rather than checking one-by-one? 

```ruby
permissions_keys.any? { |key| changed.key?(key) }

private

  def permissions_keys
    ['edit_users', 'edit_groups', 'read_users', 'read_groups']
  end
```

OTOH, the way it's currently written is easy to understand, and I'm not sure there's any performance benefit to using the iterator method here since the method as currently written should return true as soon as the first expression returns true, right?

Reply:
Yeah, I think it's easier to read as written.